src/meshapi/views/forms.py

```diff
@@ -88,6 +108,11 @@ def join_form(request):
             {"detail": "Your address could not be validated."}, status=status.HTTP_500_INTERNAL_SERVER_ERROR
         )
 
+    # No need to lock all DB objects since we don't modify these tables,
```

Comment:
We don't? Just to be clear, do you mean to say that all we do is append? What if we get a response from the same Member with different info? IIRC, we update their information, don't we?

Reply:
Currently, we do not update their information, to prevent griefing. We have an open issue about this: #93